app/src/main/java/fr/publicissapient/planningpoker/model/CardSuitType.kt

```diff
@@ -0,0 +1,7 @@
+package fr.publicissapient.planningpoker.model
+
+sealed class CardSuitType {
+    companion object {
+        const val FIBONACCI_RED = "FIBONACCI_RED"
+    }
+}
```

Comment:
If we are using sealed class, I think it's better this way:
```suggestion
sealed class CardSuitType {
    object Fibonacci :  CardSuitType()
    object TshirtSize : CardSuitType()
}
```

Reply:
Here a discriminent is needed in order to select appropriate type of card, Compose uses String only 🤔

Reply:
Red is removed!